docs/src/apps/rent.md

```diff
@@ -1 +1,46 @@
-# Accounts and Rent
+# Storage Rent for Accounts
```

Comment:
@CriesofCarrots Finally, I've added the content. Could you specifically reivew this file to check whether this meets being a straightforward, functional, user-driven introduction to rent with topic of ` developers and other users who just want to know when and how and how much rent will be assessed to their accounts.`. :) I had that in mind while writing this.